test/renderer_test.rb

```diff
@@ -55,6 +55,14 @@ class RendererTest < NicePartials::Test
     assert_css "p", text: "UPCASED"
   end
 
+  test "TestController.render partial that declares helper methods" do
+    TestController.renderer.render partial: "partial_with_helpers" do
```

Comment:
@kaspth currently, this PR is aspirational. This test fails with the stack trace provided in the PR description. I haven't had much success troubleshooting, but I have the sense that it's related to how the monkey patch is (or isn't) being applied to the `ActionView::Base` instance available to `renderer.render` calls.